docs/pages/Guides/UpdatingCompose.md

```diff
@@ -0,0 +1,94 @@
+
+Here is an example of a more common use case:
+
+```jsx
+  const CustomText = Text.customize({ fontSize: 'header', color: 'hotpink' });
+  const ComposedButton = Button.compose({
+    slots: {
+      root: View,
+      icon: Icon,
+      content: CustomText,
+    },
+    slotProps: {
+      content: {
+        // Only overrides the style prop of the content slot - other props and slots are left alone
+        style: { marginTop: -1, marginBottom: 1, marginStart: 0, marginEnd: -2 },
+      },
+    },
+  });
+
+```
+
+## Moving from old to new
+
+### DisplayName
+
+The `displayName` property stays the same between the two versions.
+
+### Replacing slots
+
+The general structure of the slots property stays the same between the old and new versions, however the slots themselves may have changed between the old and new version of a component.
+
+### Passing tokens and other styling to a slot
+
+The old API took care of this through the `settings` and `styles` properties. The new API does both under the `slotProps` property. The `slotProps` property takes an object with properties corresponding to the slot names of the control, and the properties can either be objects of the prop type for the slot, or they can be a mapped to a helper function (`buildProps`) which will generate the prop object based on the control's token values and the theme:
+
+```ts
+import { Theme, buildProps } from '@fluentui-react-native/framework';
+
+slotProps: {
+    // Object example
+    content: {
+      style: { marginTop: -1, marginBottom: 1, marginStart: 0, marginEnd: -2 },
+    },
+    // Function example
+    root: buildProps(
+      (tokens: TTokens, theme: Theme) => ({
+        style: {
+          display: 'flex',
+          alignItems: 'center',
+          flexDirection: 'row',
+          alignSelf: 'flex-start',
+          justifyContent: 'center',
+          width: tokens.width,
+          backgroundColor: tokens.backgroundColor,
+          borderColor: theme.colors.neutralStroke1,
+        },
+      }),
+      ['backgroundColor', 'width'], // Keys to be used for caching. These are the names of the tokens that the props depend on
+    ),
+}
+```
+
+The `slotProps` property only takes care of the properties of `settings` that are named after slots. For migrating the token portions, you can add a `token` propery with the customized tokens defined. The contnet of the `token` property would be what you would pass into the customize API. See guidance on how to migrate the customize API [here](./UpdatingCustomize.md) for formatting guidance.
+
+### Replacing `usePrepareProps`
+
+There is no direct equivalent for `usePrepareProps` in the new compose API. Migrating from `usePrepareProps` can be difficult, so if there are any issues please open an issue on our repo and we will be happy to help.
```

Comment:
People might wonder where to put their hooks, which is what usePrepareProps was mostly used for.  I know it's not directly transferrable, but it may be worth mentioning that hooks should go outside of the closure returned by the render function

Reply:
Thanks, added a sentence here to mention that.